### DISCUSSION TITLE
[WeakAuras] Show only when three conditions at same time

### DISCUSSION THREAD

**Discussion by unitscan** [chromiecraft/chromiecraft#4527] [Guides]
These are my triggers to remind me to use Drain Soul
Trigger 1: Player/Unit info —› Health: Unit: Boss;  Health(%) <= 25.
Trigger 2: Player/Unit info —› Unit Characteristic: Hostility: Hostile; Character type: Non-player character; Attackable.
Trigger 3: Player/Unit info —› Unit Characteristic: Talent selected: Haunt.

This WA is always displayed due to Trigger 3. How can I make this WA shows only when ALL the three conditions are true?

**Comment by Cooties-cc** [comment]
Perhaps convert the logic of the triggers to not be an OR logic gate. Make the conditions of the triggers to be AND logic, which means all 3 have to be true for it to display. In WA trigger options, this would mean to switch req activation to "all" instead of "any". 

Also I think you may have to swap out the "boss" trigger, since iirc chromie server coding doesn't label bosses like this (at least in vanilla). Zero of my WA using a unit trigger with boss worked, and instead was using focus target. If you do use focus as the main trigger to watch boss %hp, then you can actually drop the other 2 triggers iirc. This way you just have boss as focus target, and when it drops to 25% or lower, the WA triggers. No need to track spec or hostility, though will have to manually focus each boss.

**Comment by Icebeam666** [comment]
Delete trigger 3, make it the load condition instead of a trigger NO PROBLEM : )
